docs/configuration.v2alpha1.md

```diff
@@ -35,6 +35,12 @@ spec:
 | Parameter | Description |
 | --------- | ----------- |
 | features.admissionController.agentCommunicationMode | AgentCommunicationMode corresponds to the mode used by the Datadog application libraries to communicate with the Agent. It can be "hostip", "service", or "socket". |
+| features.admissionController.agentSidecarInjection.clusterAgentCommunicationEnabled | ClusterAgentCommunicationEnabled enables communication between Agent sidecars and the Cluster Agent. default : true |
+| features.admissionController.agentSidecarInjection.enabled | Enabled enables Sidecar injections. Default: true |
+| features.admissionController.agentSidecarInjection.imageName | ImageName overrides the default agent image name for the Agent sidecar. |
+| features.admissionController.agentSidecarInjection.imageTag | ImageTag overrides the default agent image tag for the Agent sidecar. |
+| features.admissionController.agentSidecarInjection.provider | Provider defines the Currently only "fargate" is supported If not specified, It will automatically set to "fargate" To use the feature in other environments (including local testing) omit the config. ref: https://docs.datadoghq.com/integrations/eks_fargate |
```

Comment:
```suggestion
| features.admissionController.agentSidecarInjection.provider | Provider defines the provider for the Datadog Agent sidecar injection. Currently, only "fargate" is supported. If not specified, it is automatically be set to "fargate". To use the feature in other environments (including local testing) omit the config. See also: https://docs.datadoghq.com/integrations/eks_fargate |
```
Please let me know if this is accurate. It seems like some sentences were merged together, so it wasn't clear to me.